blog: Add blog post announcing v1.1 RC2 (#1317)
slsa-framework/slsa: docs/_posts/2025-04-04-slsa-v1.1-rc2.md

---
title: Announcing SLSA v1.1 Release Candidate 2
author: "SLSA Community"
is_guest_post: false
---

> 

Author: Sounds good to me. Thanks!

---
title: Announcing SLSA v1.1 Release Candidate 2
author: "SLSA Community"
is_guest_post: false
---

Since the publication of [SLSA 1.0](/spec/v1.0/) back in April 2023, the
SLSA specification community has been busy developing several new tracks

> covering areas such as source and build environment but other improvements
> were made to the specification that were deemed worth publishing without
> waiting for those additions.

Reviewer: ```suggestion
covering areas such as source and build environment. While we are excited about
these other specification additions, we wanted to publishing without waiting for those
additions to be finalized.
```

Author: Sorry but this suggested text is broken.

Reviewer: What do you mean it is broken? The proposed text is
> covering areas such as source and build environment. While we are excited about
these other specification additions, we wanted to publishing without waiting for those
additions to be finalized.

Author: Ok, I'm not a native speaker but I don't think this is grammatically correct, is it? "we wanted to publishing without waiting".
And what does "these other specification additions" refer to when you haven't talked about the "other improvements" which you deleted.

Reviewer: Ah, you are right. Apologies. Clearly my brain wasn't thinking clearly yesterday as this was at least the second suggestion that was grammatically incorrect. 😓 

```suggestion
covering areas such as source and build environment. While we are excited about
these other specification additions, we wanted to publish updates without waiting for
those additions to be finalized.
```

Author: Ok, I think this is still not quite right but I get the gist of it and changed the text along the lines you suggested.

Reviewer: I will resolve this and look at the current text.

To that end [SLSA Version 1.1 Release Candidate 1 (RC1)](/spec/v1.1-rc1/)
was published in August 2024. Shortly afterwards, an astute reader pointed
out that the new threat model page had several unfinished sections, which
forced us to go through another round of edits. While this took longer than
we had anticipated (mostly due to the fact that the main contributors were
focusing on developing the new tracks), we are pleased to announce that

> [Version 1.1 RC2](/spec/v1.1-rc2/) is now available for review.

Reviewer: I am not sure that we need this paragraph.

Author: I'm open to making some changes but I think it's worth explaining why it took so long.

Author: I revised this.

Reviewer: I will resolve this and look at the current text.

> This new version brings several changes aimed at enhancing the clarity and

Reviewer: If we remove the previous paragraph, we still need to introduce the current version.
```suggestion
[Version 1.1 RC2](/spec/v1.1-rc2/) brings several changes aimed at enhancing the clarity and
```

This new version brings several changes aimed at enhancing the clarity and

> usability of the specification. In particular, this update refines the threat
> model and possible mitigations, clarifies the role of attestation format
> schemas and procedure for verifying Verification Summary Attestations (VSA),
> and adds verifier metadata to the VSA format as well as recommends to use
> cryptographic digests in VSA policies. Please, refer to the [What's
> new](/spec/v1.1-rc2/whats-new) section for further details.

Reviewer 2: ```suggestion
usability of the specification. In particular, this update refines the threat
model and possible mitigations, clarifies the role of attestation format
schemas and procedure for verifying Verification Summary Attestations (VSA),
and adds verifier metadata to the VSA. Please, refer to the [What's
new](/spec/v1.1-rc2/whats-new) section for further details.
```

I don't think we need the detail about VSA 'policies', it all rolls up into the "procedure for verifying" _and_ the change doesn't actually mention _cryptographic_ digests, just that the digest _field_ is set to something (it can be non-cryptographic digests).

> usability of the specification. In particular, this update brings several
> refinements to the threat model and possible mitigations, some
> clarification to the role of attestation format schemas, a procedure for
> verifying Verification Summary Attestations (VSA), provision to add
> verifier metadata into the VSA format, and a recommendation to use
> cryptographic digests in VSA policies. Please, refer to the [What's
> new](/spec/v1.1-rc2/whats-new) section for further details.

Reviewer: Is this wording any better?
```suggestion
usability of the specification. In particular, this update refines the threat
model and possible mitigations, clarifies the role of attestation format
schemas and procedure for verifying Verification Summary Attestations (VSA),
and adds verifier metadata to the VSA format as well as recommends to use
cryptographic digests in VSA policies. Please, refer to the [What's
new](/spec/v1.1-rc2/whats-new) section for further details.
```

It is worth noting that none of the changes brought in SLSA 1.1 affect the

> compliance of an existing SLSA 1.0 implementation.

Reviewer: Is this effectively just to say that the changes are backwards compatible?

Author: I was first going to go further and say that the normative part of the spec has not changed but I think the VSA related changes are normative so indeed this is maybe a long way to say that the changes are backwards compatible.

Reviewer 2: Yeah let's say it's backwards compatible explicitly.

Author: Done.
